src/store/index.js

```diff
@@ -27,4 +27,17 @@ export default new Vuex.Store({
       commit("setScreenings", data);
     },
   },
+  getters: {
+    movies: (state) => state.movies,
+    screenings: (state) => state.screenings,
+    genres: (state) => {
+      const genres = [];
+      state.movies.forEach((movie) => {
+        if (!genres.includes(movie.genre.name)) {
+          genres.push(movie.genre.name);
+        }
+      });
+      return genres;
+    },
```

Comment:
Since it's simple array of strings you can combine spread operator and `Set` constructor. This way you'll eliminate additional checks.
```
genres: (state) => [...new Set(state.movies.map((movie) => movie.genre.name ))]
```